tmdsimpy/jax/nlforces/elastic_dry_fric_3d.py

```diff
@@ -0,0 +1,823 @@
+# Standard imports
+import numpy as np
+import warnings
+
+# JAX imports
+import jax
+import jax.numpy as jnp
+
+# Decoractions for Partial compilation
+from functools import partial
+
+# Imports of Custom Functions and Classes
+from ... import harmonic_utils as hutils
```

Comment:
The old import of `harmonic_utils` is deprecated. It should work with:

`from ...utils import harmonic as hutils`